src/test/java/org/fogbowcloud/manager/requests/api/local/http/ImageRequestControllerTest.java

```diff
@@ -0,0 +1,164 @@
+package org.fogbowcloud.manager.requests.api.local.http;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import org.fogbowcloud.manager.api.http.ImageRequestController;
+import org.fogbowcloud.manager.core.ApplicationFacade;
+import org.fogbowcloud.manager.core.exceptions.FogbowManagerException;
+import org.fogbowcloud.manager.core.exceptions.InstanceNotFoundException;
+import org.fogbowcloud.manager.core.models.images.Image;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.*;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockRunnerDelegate(SpringRunner.class)
+@WebMvcTest(value = ImageRequestController.class, secure = false)
+@PrepareForTest(ApplicationFacade.class)
+public class ImageRequestControllerTest {
+
+    private final String IMAGE_ENDPOINT = "/" + ImageRequestController.IMAGE_ENDPOINT;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    private ApplicationFacade facade;
+
+    @Before
+    public void setUp() throws FogbowManagerException {
+        this.facade = spy(ApplicationFacade.class);
+        PowerMockito.mockStatic(ApplicationFacade.class);
+        given(ApplicationFacade.getInstance()).willReturn(this.facade);
```

Comment:
follow the convention. Do not use static imports. I guess there are more calls like that. Please, check this out.

Reply:
Fixed!